chain/ethereum/src/network_indexer/ethereum.graphql

```diff
@@ -62,25 +62,126 @@ type Block @entity {
   """The number of ommers (AKA uncles) associated with this block."""
   ommerCount: Int!
 
-  # """
-  # A list of ommer (AKA uncle) blocks associated with this block.
-  # """
+  """
+  A list of ommer (AKA uncle) blocks associated with this block.
+  """
   ommers: [Block]!
 
   """The keccak256 hash of all the ommers (AKA uncles) associated with this block."""
   ommerHash: Bytes!
 
-  # """
-  # A list of transactions associated with this block.
-  # """
-  # transactions: [Transaction!]
+  """
+  A list of transactions associated with this block.
+  """
+  transactions: [Transaction!]
+
+  """The number of logs emitted in this block."""
+  logCount: Int!
 
-  # """The logs emitted in this block."""
-  # logs: [Log!]!
+  """The logs emitted in this block."""
+  logs: [Log!]
 
   """Size of the block in bytes."""
   size: BigInt
 
   """Seal fields."""
   sealFields: [Bytes!]!
 }
+
+""" An Ethereum transaction. """
+type Transaction @entity {
+  id: ID!
+
+  """ The hash of this transaction. """
+  hash: Bytes!
+
+  """ The nonce of the account this transaction was generated with. """
+  nonce: BigInt!
+
+  """
+  The index of this transaction in the parent block.
+  This will be null if the transaction has not yet been mined.
+  """
+  index: BigInt
+
+  """ The account that sent this transaction - this will always be an externally owned account. """
+  from: Bytes!
+
+  """ The account the transaction was sent to. This is null for contract-creating transactions. """
+  to: Bytes
+
+  """ The value, in wei, sent along with this transaction. """
+  value: BigInt!
+
+  """ The price offered to miners for gas, in wei per unit. """
+  gasPrice: BigInt!
+
+  """ The maximum amount of gas this transaction can consume. """
+  gas: BigInt!
+
+  #  """ The data supplied to the target of the transaction. """
+  #  inputData: Bytes!
```

Comment:
Are we not able to include this data?

Reply:
There are some blocks with exceedingly large `inputData` which causes insert errors since the btree index row size limits are hit. 
My thinking is the way past the size error is with custom indices, so the inputData field can be set as one that does not get indexed.